class.jetpack-jitm.php

```diff
@@ -227,17 +242,34 @@ static function get_messages( $message_path, $query ) {
 
 		// build our jitm request
 		$path = add_query_arg( array(
-			'external_user_id'     => urlencode_deep( $user->ID ),
-			'user_roles'           => urlencode_deep( implode( ',', $user->roles ) ),
-			'query_string'         => urlencode_deep( $query ),
+			'external_user_id' => urlencode_deep( $user->ID ),
+			'user_roles'       => urlencode_deep( implode( ',', $user->roles ) ),
+			'query_string'     => urlencode_deep( $query ),
 		), sprintf( '/sites/%d/jitm/%s', $site_id, $message_path ) );
 
 		// attempt to get from cache
-		$envelopes  = get_transient( 'jetpack_jitm_' . substr( md5( $path ), 0, 31 ) );
+		$envelopes = get_transient( 'jetpack_jitm_' . substr( md5( $path ), 0, 31 ) );
 
 		// if something is in the cache and it was put in the cache after the last sync we care about, use it
-		$last_sync = (int) get_transient( 'jetpack_last_plugin_sync' );
-		$from_cache = $envelopes && $last_sync > 0 && $last_sync < $envelopes['last_response_time'];
+		$use_cache = false;
+
+		/**
+		 * Filter to turn off jitm caching
+		 *
+		 * @since 5.4.0
+		 *
+		 * @param bool true Whether to cache just in time messages
+		 */
```

Comment:
It may trip our parser if we have a filter definition for the same filter 3 times. Could you only keep one docblock, and refer to that one for the other instances where `jetpack_just_in_time_msg_cache` appears?

Thanks!